api/src/lib/email.ts

```diff
@@ -956,20 +969,18 @@ export const incrementalAriIngestReport = async (options: {
         </html>
     `;
     const text = `
-        Incremental ARI ingest run completed in ${options.durationSeconds} seconds.
-        ARIs checked: ${options.checkedCount}.
-        Publications created: ${options.createdCount}.
-        Publications updated: ${options.updatedCount}.
-        ${
-            options.unrecognisedDepartments.length
-                ? 'Unrecognised departments: "' + options.unrecognisedDepartments.join('", "') + '".'
-                : ''
-        }
-        ${
-            options.unrecognisedTopics.length
-                ? 'Unrecognised topics: "' + options.unrecognisedTopics.join('", "') + '".'
-                : ''
-        }`;
+${intro}
```

Comment:
Formatting seems to be off here

Reply:
I did this on purpose, the default formatting had it indented which reads better, but the problem was that the indentation carried through into the string and into the email output.

Reply:
:+1: Didn't think of that